plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/github.ts

```diff
@@ -225,6 +228,16 @@ export function createPublishGithubAction(options: {
         logger: ctx.logger,
       });
 
+      try {
+        await enableBranchProtectionOnDefaultRepoBranch({
+          owner,
+          client,
+          repoName: newRepo.name,
+        });
+      } catch (e) {
+        throw new Error(`Failed to add branch protection to '${name}', ${e}`);
```

Comment:
is this accidentally the global `module.name`? i don't see a name variable above - did you mean `repo` or `newRepo.name`?

Reply:
Great catch! Thanks!